### PR TITLE
Added buttons in navbar for sign in, sign up and sign out

### PR DESCRIPTION
**Pull request by UsmanAhmad169**
Added user sign in, sign out and sign up buttons in navbar.

### CODE REVIEW

**Comment by UsmanAhmad169** [comment]
cycle 0

File: config/locales/en.yml [hunk 1]
```diff
@@ -34,6 +34,8 @@ en:
     email: "email"
     update: "Update"
   users:
+    sign_in: "Sign in"
```
Comment (muhammad-basil) [inline config/locales/en.yml]: use single quote

File: app/views/layouts/_navbar.html.erb [hunk 2]
```diff
@@ -18,5 +18,24 @@
...
+        </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+          <a class="dropdown-item" href="#">Action</a>
+          <a class="dropdown-item" href="#">Another action</a>
+          <div class="dropdown-divider"></div>
+          <%= link_to t('users.sign_out'), destroy_user_session_path, method: :delete, class: "dropdown-item" %>
+        </div>
+      </li>
+      <% else %>
+      <%= link_to t('users.sign_in'), new_user_session_path, class: "btn btn-outline-secondary my-1 mr-md-2" %>
```
Comment (muhammad-basil) [inline app/views/layouts/_navbar.html.erb]: use single quote in class attribute. Make this consistent throughout this file

**Comment by muhammad-basil** [comment]
Cycle 1

File: app/views/layouts/_navbar.html.erb (hunk 2 above)
Comment (UsmanAhmad169) [reply app/views/layouts/_navbar.html.erb]: ok

File: app/views/layouts/_navbar.html.erb [hunk 3]
```diff
@@ -18,5 +18,24 @@
...
     </ul>
+
+    <ul class="navbar-nav ml-auto">
+      <% if user_signed_in? %>
+      <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          <%= current_user.email %>
+        </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+          <a class="dropdown-item" href="#">Action</a>
```
Comment (muhammad-basil) [inline app/views/layouts/_navbar.html.erb]: remove dummy anchor tags

File: config/locales/en.yml (hunk 1 above)
Comment (UsmanAhmad169) [reply config/locales/en.yml]: done

File: app/views/layouts/_navbar.html.erb (hunk 3 above)
Comment (UsmanAhmad169) [reply app/views/layouts/_navbar.html.erb]: done

File: app/views/layouts/_navbar.html.erb [hunk 4]
```diff
@@ -18,5 +18,21 @@
...
+        <a class="nav-link dropdown-toggle" href="#" id="user_profile_dropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          <%= current_user.email %>
+        </a>
+        <div class="dropdown-menu" aria-labelledby="user_profile_dropdown">
+          <%= link_to t('users.sign_out'), destroy_user_session_path, method: :delete, class: 'dropdown-item' %>
+        </div>
+      </li>
+      <% else %>
+      <%= link_to t('users.sign_in'), new_user_session_path, class: 'btn btn-outline-secondary my-1 mr-md-2' %>
+      <%= link_to t('users.sign_up'), new_user_registration_path, class: 'btn btn-outline-info my-1' %>
```
Comment (muhammad-basil) [inline app/views/layouts/_navbar.html.erb]: indentation

Comment (UsmanAhmad169) [reply app/views/layouts/_navbar.html.erb]: done

**Pull request merged**